extension/src/popup/components/accountHistory/TransactionDetail/index.tsx

```diff
@@ -72,12 +73,15 @@ export const TransactionDetail = ({
 
   const { t } = useTranslation();
 
-  const { assetDomain } = useAssetDomain({
+  const { assetDomain, error: assetError } = useAssetDomain({
```

Comment:
@piyalbasu A good follow up here would be to handle the asset error more explicitly imo. If we fail to fetch an asset domain when we have an asset issuer, in `AssetNetworkInfo` we will incorrectly display these as `Stellar Lumens`.

Reply:
Sorry correction, it just shows an empty space using the empty string. Still looks a bit strange but maybe not as urgent as the mislabeling I thought would happen.

Reply:
Great - yeah, a slightly less optimal UI in an error case is preferable to incorrect info

Reply:
but we can follow up and fix this